src/Components/Facility/FacilityHome.tsx

```diff
@@ -106,7 +106,7 @@ export const FacilityHome = (props: any) => {
   const editCoverImageTooltip = hasPermissionToEditCoverImage && (
     <div
       id="facility-coverimage"
-      className="absolute right-0 top-0 z-10 flex h-full w-full flex-col items-center justify-center bg-black text-sm text-gray-300 opacity-0 transition-opacity hover:opacity-60 md:h-[88px]"
+      className="absolute right-0 top-0 z-10 flex h-full w-full flex-col items-center justify-center rounded-t-lg bg-black text-sm text-gray-300 opacity-0 transition-[opacity] hover:opacity-60 md:h-[88px]"
```

Comment:
@kshitijv256 please double check `transistion-[opacity]` class

![image](https://github.com/coronasafe/care_fe/assets/29787772/9d14b01f-1fc9-41bc-a38a-fd2710fc9299)

Reply:
@khavinshankar 
I missed it as I was trying out other transitions there, changed it back to `transition-opacity`